bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -317,6 +317,13 @@ spec:
           - get
           - list
           - watch
+        - apiGroups:
+          - machineconfiguration.openshift.io
+          resources:
+          - machineconfigs
+          verbs:
+          - list
+          - watch
```

Comment:
I was getting errors in the WMCO log around failing to get the machine config resource because it was unable to watch it. I didn't really understand why I was getting the error honestly, but giving watch permissions "fixed" it.